### DISCUSSION TITLE
vless+xtls在Header Type为NONE时启动失败"Unknown plugin xtls-plugin"

### DISCUSSION THREAD

**Discussion by 10ta** [SagerNet/SagerNet#90] [Q&A] [answered]
**Describe the bug**
如题 推测和#63是同一问题

**To Reproduce**
不论是手动配置vless 或从v2rayNG导入可成功连接使用的xtls配置,
按上述操作启动连接即可复现

**Logcat**
Export it yourself :)

**Expected behavior**
能成功启动并连接成功

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Pixel 4
 - OS: Android 11
 - Version 0.1-rc09

**Additional context**

**Comment by nekohasekai** [comment, marked as answer]
检查权限

**Reply by 10ta** [reply]
权限是全开的,  且当Header Type为HTTP时启动成功. 
排除权限问题

**Reply by nekohasekai** [reply]
xtls 只支持tcp none, 所以其他设置都没有走 xtls.

**Reply by nekohasekai** [reply]
这就排除了? 你看得懂代码还是

**Reply by 10ta** [reply]
我说了权限是全开的没看到吗? 你的app还需要什么神奇的权限才能使用xtls ?

**Reply by nekohasekai** [reply]
自启动, 允许唤醒之类
